holmes/core/supabase_dal.py

```
@@ -518,10 +528,13 @@ def get_workload_issues(self, resource: dict, since_hours: float) -> List[str]:
                 self.client.table(EVIDENCE_TABLE)
                 .select("data, enrichment_type")
                 .in_("issue_id", unique_issues)
+                .not_.in_("enrichment_type", ENRICHMENT_BLACKLIST)
                 .execute()
             )
 
-            return self.extract_relevant_issues(res)
+            relevant_issues = self.extract_relevant_issues(res)
+            truncate_evidences_entities_if_necessary(relevant_issues)
+            return relevant_issues
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)